### DISCUSSION TITLE
rtp1 não está a funcionar bem

### DISCUSSION THREAD

**Discussion by lfm99** [LITUATUI/M3UPT#28] [General]
os canais da RTP estão a funcionar com muito "buffering", creio que esta situação é recente.
Alguma ideia? será que teremos que explorar forma de usar o rtpClean1HD.smil ?

**Comment by LITUATUI** [comment]
É estranho porque este problema só ocorre no Kodi, no Perfect Player e no ProgTV não.

Isto já tinha acontecido uma vez, acho que voltou ao normal passado um tempo sem eu ter de mudar alguma coisa na lista.

**Reply by lfm99** [reply]
pois há poucos dias atrás, quer no VLC (PC) quer no kodi e Mxplayer (android) tudo funcionava bem. Hoje não consigo ver o canal 1 em nenhum destes players. Não sei se é algo temporário, mas estou desconfiado que não, pois o m3u que atualmente obtenho tem a palavra "Clean" na endereço smil... e com esse não funciona em nenhum player para já...

**Reply by LITUATUI** [reply]
O addon [RTP Play](https://kodi.wiki/view/Add-on:RTP_Play) para o Kodi está a funcionar bem. 

É questão de perder algum tempo a ver os logs do Kodi. O problema poderá estar no user-agent.

**Reply by LITUATUI** [reply]
Bem, modifiquei a lista para os canais RTP voltarem a funcionar no Kodi. Vamos ver quanto tempo dura.

**Reply by lfm99** [reply]
Boa! o url mudou um bocado. Sim, vamos ver até quando ;) abr.
